src/components/VContentReport/VReportDescForm.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <label class="other-form block" for="description">
+    <span class="flex flex-row justify-between items-center">
+      <span>{{ $t('media-details.content-report.form.other.note') }}</span>
+      <span>{{
+        $t(
+          isRequired
+            ? 'media-details.content-report.form.required'
+            : 'media-details.content-report.form.optional'
+        )
+      }}</span>
+    </span>
+    <textarea
+      id="description"
+      v-model="text"
+      class="h-20 w-full border border-dark-charcoal-20 placeholder-dark-charcoal-70 mt-2 p-2"
+      :placeholder="$t('media-details.content-report.form.other.minimum')"
```

Comment:
We should change the placeholder to something different when `isRequired` is `false`, otherwise the "Optional" label and the placeholder text kind of conflict with each other.

Should we also remove the `minlength` for the mature report? I could see someone trying to just enter "gore" or "porn" in the description box and being confused why they have to write an arbitrary 20 character sentence just to use a one-word description of what makes the content "mature" in their view.

Reply:
I hope the changes in 7fff77b should fix this.